### ISSUE TITLE
Accessibility view: all actions have a bell icon

### ISSUE THREAD

**Issue by andreamah** [microsoft/vscode#191661]
Testing #191344

This is probably intentional, but I notice that all notification actions use a bell icon. Have you considered something like a sub-menu for the different options instead of repeating an icon?

![Image](https://github.com/microsoft/vscode/assets/31675041/631f5fa6-4ac1-4850-93ea-4077e3c4b321)

**Comment by meganrogge** [comment]
We want to have these in a top level menu so that they can be navigated using just arrow keys as that's easier for screen reader users. Without knowing what each action is (which we cannot know as they could be coming from extensions), we have to go with the default here, the bell.

**Comment by VSCodeTriageBot** [comment]
The described behavior is how it is expected to work. If you disagree, please explain what is expected and what is not in more detail. See also our [issue reporting guidelines](https://aka.ms/vscodeissuereporting).

Happy Coding!
